content/java/concepts/math-methods/terms/tan/tan.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.tan()'
+Description: 'Returns the tangent of an angle given in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Function'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.tan()`** method returns tangent of an angle given in radians. The function accepts all real numbers *except* the values where `cos(x)` is equal to 0  
+
+## Syntax
+
+```pseudo
+Math.tan(double x)
+```
+Here `x` is the argument passed to the function. It accepts all real numbers. 
```

Comment:
Place a comma after "Here". The word "it" in the second sentence is unclear as to what it is referring to. Consider this sentence instead:

"`Math.tan()` accepts all real numbers as arguments."